tests/testthat/test-add_gloabl_p.tbl_uvregression.R

```diff
@@ -0,0 +1,76 @@
+test_that("add_n.tbl_regression() works", {
+  tbl <- tbl_uvregression(
+        trial,
+        method = glm,
+        include = c(trt, grade, age),
+        y = response,
+        method.args = list(family = binomial),
+        exponentiate = TRUE
+    )
+
+  # total N added to table is accurate
+  expect_error(
+    res <- tbl |> add_global_p(),
+    NA
+  )
+
+  expect_snapshot(
+    res |> as.data.frame()
+  )
+})
+
+
+test_that("no errors/warnings with standard use after tbl_uvregression", {
```

Comment:
@ddsjoberg is there a reason why the [this](https://github.com/ddsjoberg/gtsummary/blob/main/tests/testthat/test-add_global_p.R)  file went away? I just ported over what was preexisting for now